tests/test_examples.py

```diff
@@ -0,0 +1,10 @@
+from pathlib import Path
```

Comment:
Could you copy the license + copyright statement from the other files to here as well please? I know it is not the most pretty, but I would prefer to have the files consistent.

Reply:
Added in next commit.